src/main/scala/org/scalajs/tools/tsimporter/Importer.scala

```diff
@@ -280,6 +280,14 @@ class Importer(val output: java.io.PrintWriter) {
         TypeRef.Singleton(QualifiedName((expr.qualifier :+ expr.name).map(
             ident => Name(ident.name)): _*))
 
+      case TupleType(targs) =>
+        if (targs.forall( _ == targs.head)) {
+          TypeRef(QualifiedName.Array, List(typeToScala(targs.head)))
```

Comment:
Maybe we should actually emit `js.Array[_ <: T]` instead of `js.Array[T]`, because there is some intention of being covariant, there, even though TypeScript does not really have covariance.